python/ray/tune/analysis/experiment_analysis.py

```diff
@@ -757,14 +764,28 @@ def runner_data(self) -> Dict:
 
     def _get_trial_paths(self) -> List[str]:
         if self.trials:
-            _trial_paths = [t.logdir for t in self.trials]
+            # Get the relative paths from the trials to allow
+            # for changes in the self._local_base_dir.
+            _trial_paths = [
+                str(self._local_base_dir.joinpath(t.rel_logdir))
+                if t.rel_logdir
+                else t.rel_logdir
+                for t in self.trials
+            ]
         else:
             logger.info(
                 "No `self.trials`. Drawing logdirs from checkpoint "
                 "file. This may result in some information that is "
                 "out of sync, as checkpointing is periodic."
             )
-            _trial_paths = [checkpoint["logdir"] for checkpoint in self._checkpoints]
+
+            _trial_paths = [
+                str(self._local_base_dir.joinpath(checkpoint["rel_logdir"]))
+                if checkpoint["rel_logdir"]
+                else checkpoint["rel_logdir"]
```

Comment:
Same here

```suggestion
                else checkpoint["logdir"]
```

Reply:
In this case we should turn the if into `if checkpoint.get("rel_logdir", None)` for backwards compat

Reply:
I have used now `relative_logdir` and `logdir` in the code blocks above.